javascripts/custom.js

```diff
@@ -0,0 +1,24 @@
+jQuery(function () {
+    const $ = jQuery;
+    $(".gallery-nav li a").click((e) => {
+        const cat = $(e.target).data("cat");
+        if (cat === "tous") return;
+
+        const el = $(`.gallery .${cat}`).first();
+        const h = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
+        const p = el.position();
+
+        if (p.top > h) {
+
+            let scrollTime = Math.round((p.top - h) / 3);
+            scrollTime = Math.min(scrollTime, 800);
```

Comment:
Swap min and max

Reply:
what do you mean? you don't want more than 800 ms and not less than 300

Reply:
Sorry, read that too fast 🤦‍♂ After testing it, 300ms seems a bit fast

Reply:
I'll let you decide of a decent minimal time :)